examples/cp-all-in-one/docker-compose.yml

```diff
@@ -11,16 +11,23 @@ services:
       ZOOKEEPER_TICK_TIME: 2000
 
   broker:
-    image: confluentinc/cp-kafka
+    image: confluentinc/cp-enterprise-kafka
     hostname: broker
     depends_on:
       - zookeeper
     ports:
-      - '9092'
+      - "9092:9092"
     environment:
       KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
       KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT://broker:9092'
+      KAFKA_METRIC_REPORTERS: io.confluent.metrics.reporter.ConfluentMetricsReporter
+      CONFLUENT_METRICS_REPORTER_BOOTSTRAP_SERVERS: broker:9092
+      CONFLUENT_METRICS_REPORTER_ZOOKEEPER_CONNECT: zookeeper:2181
+      CONFLUENT_METRICS_REPORTER_TOPIC_REPLICAS: 1
+      CONFLUENT_METRICS_ENABLE: 'true'
+      CONFLUENT_SUPPORT_CUSTOMER_ID: 'anonymous'
+      CONFLUENT_METRICS_REPORTER_PUBLISH_MS: 1000
```

Comment:
that last line specifically - "CONFLUENT_METRICS_REPORTER_PUBLISH_MS" is something i've never needed to use though, i suspect it will have some negative side effect if left running with that param for too long